src/App.tsx

```diff
@@ -29,6 +30,22 @@ function App() {
 					<Route path="/register" element={<RegisterPage />} />
 					<Route path="/login" element={<LoginPage />} />
 					<Route path="/cart" element={<CartPage />} />
+					<Route
+						path="/electronics"
+						element={<ProductCategoryPage category="electronics" />}
+					/>
```

Comment:
The same as above, move categories names to enum `src/constants/categories.ts`.
```suggestion
			export enum Categories {
			    Jewelry = "jewelry",
			    // ... etc
			}